components/brave_wallet_ui/BUILD.gn

```diff
@@ -4,7 +4,6 @@
 # you can obtain one at http://mozilla.org/MPL/2.0/.
 
 import("//brave/components/common/typescript.gni")
-import("//chrome/common/features.gni")
```

Comment:
please create a separate PR for removing `import("//chrome/common/features.gni")` and rebase this one on top of it